### PR TITLE
TECH-339: Migrate GlobalStyle, ImgWrapper, and SvgWrapper components to Typescript

### PR DESCRIPTION
**Pull request by vindhya**
## JIRA

https://jira.jahia.org/browse/TECH-339

## Description

Components migrated to Typescript:
* GlobalStyle
* ImgWrapper
* SvgWrapper
   * Changed the way the `viewBox` and `fill` attributes are accessed because the `.attributes` way is an old API and causes issues with types

### CODE REVIEW

File: src/components/ImgWrapper/ImgWrapper.types.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export type ImgWrapperSize = 'small' | 'default' | 'big';
+export const imgWrapperSize = ['small', 'default', 'big'];
```
Comment (ffffffelix) [inline src/components/ImgWrapper/ImgWrapper.types.ts:2]: It seems for other components (Button, Badges, Chip ...) we use the pluriel to describe the array `imgWrapperSizes`.
I don't know which convention we want to apply.

File: src/components/SvgWrapper/SvgWrapper.types.ts [hunk 2]
```diff
@@ -1,6 +1,9 @@
-import * as React from 'react';
+export type SvgWrapperSize = 'small' | 'default' | 'big';
+export const svgWrapperSize = ['small', 'default', 'big'];
```
Comment (ffffffelix) [inline src/components/SvgWrapper/SvgWrapper.types.ts:2]: Same question here, do we want to use the pluriel: `svgWrapperSizes`?

File: src/components/ImgWrapper/ImgWrapper.types.ts (hunk 1 above)
Comment (vindhya) [reply src/components/ImgWrapper/ImgWrapper.types.ts]: I updated it to be the same as the other components. Nice catch 👀

File: src/components/SvgWrapper/SvgWrapper.types.ts (hunk 2 above)
Comment (vindhya) [reply src/components/SvgWrapper/SvgWrapper.types.ts]: Updated!

**Review by ffffffelix — APPROVED** [review]

**Pull request merged**